development/tools/jobs/tester/tester.go

```diff
@@ -103,15 +103,17 @@ const (
 
 // SupportedRelease defines supported releases
 type SupportedRelease = string
+type unSupportedRelease = string
```

Comment:
Is this used somewhere?

Reply:
If not, please just delete it.

Reply:
As discussed in person: Gonna remove the 1.1 release const including all references in the go files.